### PR TITLE
Increase number of data points to write per batch

### PR DESCRIPTION
**Pull request by hakontro**
Before this we were only writing batches of 1,000 data points. This will increase this to 10,000 data points.

### CODE REVIEW

File: src/main/scala/cognite/spark/v1/DefaultSource.scala [hunk 1]
```diff
@@ -233,28 +233,33 @@ class DefaultSource
           new StringDataPointsRelationV1(config)(sqlContext)
         case _ => sys.error(s"Resource type $resourceType does not support save()")
       }
+      val batchSize = relation match {
+        case n: NumericDataPointsRelationV1 => Constants.CreateDataPointsLimit
```
Comment (exyi) [inline src/main/scala/cognite/spark/v1/DefaultSource.scala]: I think this should also apply to string data points, does it?

File: src/main/scala/cognite/spark/v1/Constants.scala [hunk 2]
```diff
@@ -5,6 +5,7 @@ import BuildInfo.BuildInfo
 import scala.concurrent.duration._
 
 object Constants {
+  val CreateDataPointsLimit = 10000
```
Comment (exyi) [inline src/main/scala/cognite/spark/v1/Constants.scala]: Why 10,000 when the limit is higher?

https://docs.cognite.com/api/v1/#operation/postMultiTimeSeriesDatapoints
>  The limit per request is 100000 datapoints.

Comment (hakontro) [reply src/main/scala/cognite/spark/v1/Constants.scala]: Ohh, good catch! I thought it was 10k from this part
>Array of DatapointsWithInternalId (object) or DatapointsWithExternalId (object) (DatapointsPostDatapoint) [ 1 .. *10000* ] items

File: src/main/scala/cognite/spark/v1/DefaultSource.scala (hunk 1 above)
Comment (hakontro) [reply src/main/scala/cognite/spark/v1/DefaultSource.scala]: It should, I just missed it. I'll implement it, thanks for pointing it out.

**Comment by wjoel** [comment]
> Before this we were only writing batches of 1,000 data points. This will increase this to 10,000 data points.

For the record, it seems to actually increase it to 100,000 (which is good).

**Pull request merged**